### PR TITLE
Display rating feature

### LINKED ISSUE #94: Display average rating of user in `ProfileActivity`
### Suggestion / Feature Request

Exactly what it says

Display 3 average ratings, overall, movies and tv shows

While calculating average ignore unrated movies or tv shows

### PR DESCRIPTION
**Pull request by armaanbadhan**
## Description

users can now see their average rating on their profile

## Issue Resolved

fixes #94 

## Screenshots 

<div>

</div>

## Checklist

Please make sure to review the following before submitting your PR: 

- [x] I have read the [contribution guidelines](https://github.com/PEC-CSS/MovieDroid/blob/main/CONTRIBUTING.md).
- [x] I have read the code of conduct.
- [x] I have reviewed my submission in detail.

### CODE REVIEW

File: app/src/main/java/com/pec_acm/moviedroid/ProfileActivity.kt [hunk 1]
```diff
@@ -29,6 +35,46 @@ class ProfileActivity : AppCompatActivity() {
...
+                        {
+                            tvCount += 1
+                            tvRate += item.personalScore
+                        }
+                        else
+                        {
+                            movieCount += 1
+                            movieRate += item.personalScore
+                        }
+                    }
```
Comment (ken1000minus7) [inline app/src/main/java/com/pec_acm/moviedroid/ProfileActivity.kt]: The logic of getting user info should be inside a function in a view model, create a `ProfileViewModel` and add this logic there and observe a live data of the user to then calculate and display the ratings

**Review by ken1000minus7 — CHANGES_REQUESTED** [review]

**Comment by ken1000minus7** [comment]
Currently I think this would display the floor value of the average, display it upto 2 decimal places

**Comment by armaanbadhan** [comment]
done!

**Review by ken1000minus7 — APPROVED** [review]
Lgtm

**Pull request merged**